examples/create-window/main.go

```diff
@@ -103,5 +105,40 @@ func main() {
 		if xerr != nil {
 			fmt.Printf("Error: %s\n", xerr)
 		}
+
+		// This is how accepting events work:
+		// The application checks what event we got
+		// (the event must be registered using either xproto.CreateWindow (see l.35) or
+		//  xproto.ChangeWindowAttributes (see l.50)).
+		// and reacts to it accordingly. All events are defined in the xproto subpackage.
+		switch ev.(type) {
+		case xproto.KeyPressEvent:
+			// See https://pkg.go.dev/github.com/jezek/xgb@v0.0.0-20210121230032-cec22bda1ce1/xproto#KeyPressEvent
+			// for documentation about a key press event.
+			kpe := ev.(xproto.KeyPressEvent)
+			fmt.Printf("Key pressed: %d", kpe.Detail)
+			// The Detail value depends on the keyboard layout,
+			// for QWERTY, q is #24.
+			if kpe.Detail == 24 {
+				return // exit on q
+			}
+		case xproto.DestroyNotifyEvent:
+			// Depending on the user's desktop environment (especially
+			// window manager), killing a window might close the
+			// client's X connection (e. g. the default Ubuntu
+			// desktop environment).
+			//
+			// If that's the case for your environment, closing this example's window
+			// will also call the underlying Go program (because closing the X
```

Comment:
Shouldn't the ```call``` be ```kill```?

Reply:
Oh yeah, typo on my side. Sorry for that